java-runtime-builder-app/pom.xml

```diff
@@ -84,6 +84,11 @@
       <artifactId>commons-io</artifactId>
       <version>2.5</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
Are you using the annotation purely for documentation? In that case, maybe just say so in the javadoc comment?